diff -ur lxpanel-0.5.12.orig/src/plugins/netstatus/netstatus-iface.c lxpanel-0.5.12.fix/src/plugins/netstatus/netstatus-iface.c
--- lxpanel-0.5.12.orig/src/plugins/netstatus/netstatus-iface.c	2012-11-03 03:10:08.000000000 +0000
+++ lxpanel-0.5.12.fix/src/plugins/netstatus/netstatus-iface.c	2013-07-18 08:56:22.016116247 +0000
@@ -408,7 +408,7 @@
   va_start (args, format);
 
   error_message = g_strdup_vprintf (format, args);
-  error = g_error_new (NETSTATUS_ERROR, code, error_message);
+  error = g_error_new (NETSTATUS_ERROR, code, "%s", error_message);
 
   dprintf (POLLING, "ERROR: %s\n", error->message);
   netstatus_iface_set_error (iface, error);
